crates/milli/src/index.rs

```diff
@@ -253,7 +252,22 @@ impl Index {
             vector_arroy,
             embedder_category_id,
             documents,
-        })
+        };
+        if this.get_version(&wtxn)?.is_none() {
+            this.put_version(
```

Comment:
this is only true for indexes that are indeed created at this point.

older indexes should be assigned `v1.12.0` abruptly or be recovered from Meilisearch.